build/Invoke-CollectionTests.ps1

```diff
@@ -104,6 +104,17 @@ process {
             $Inventory | Set-Content -Path "~/$InventoryFile" -Force
 
             bash -c $Commands
+
+            $Result = [PSCustomObject]@{
+                Success  = $?
+                ExitCode = $LASTEXITCODE
+            }
+
+            $Result
```

Comment:
I'm not sure if this is going to actually output anything... On my system it's currently erroring on line 143. I changed line 140 to include `Write-Host` and it outputs the hashtable, but without the `Write-Host` there's no output that I see.

Reply:
This branch of code is only executed in CI. Something similar _should_ be happening for Vagrant further down, though. The `throw` is only reached after it's been output... not sure why it wouldn't show.

Reply:
Amended the most recent commit to force this to show before the `throw` happens.